test/admin/testutil.js

```diff
@@ -0,0 +1,7 @@
+const stitch = require('../../src');
+
+export async function getAuthenticatedClient(apiKey) {
+  const adminClient = new stitch.Admin('http://localhost:7080');
```

Comment:
[q] Should this be made configurable?

Reply:
[q] Does there exist a follow up ticket to ensure we add the auth portion of v2 to the SDK?

Reply:
it's basically just hardcoded to match what's in the test config file in `test/configs/test_config.json`

Reply:
👍